.eslintrc

```diff
@@ -172,6 +172,27 @@
           }
         ]
       }
+    },
+    {
+      "files": ["none"],
```

Comment:
`"none"` does not have any special meaning. ESLint needs a value to be there, so I picked a string that does not match a folder name in our repo.